index.html

```diff
@@ -0,0 +1,154 @@
+    </nav>
+    <!-- second second navigation -->
+    <nav class="nav-div">
+      <div class="logo">
+        <a href="index.html"><img class=logo src="Img\pngtree-blue-bird-vector-or-color-illustration-png-image_2013004.jpg"  alt="Logo"></a>
+      </div>
+      <ul class="nav justify-content-end">
+        <li class="navy">
+          <a class="nav-link active" aria-current="page" href="index.html">Home</a>
+        </li>
+        <li class="navy">
+          <a class="nav-link active" aria-current="page" href="about.html">About</a>
+        </li>
+        <li class="navy">
+          <a class="nav-link" href="index.html#program">Join</a>
+        </li>
+        <li class="navy">
+          <a class="nav-link" href="index.html#join">Agricbusiness</a>
+        </li>
+        <li class="navy">
+          <a class="nav-link" href="index.html#sponsors">Construction</a>
+        </li>
+        <li class="navy">
+          <a class="nav-link" href="#">Consultancy</a>
+        </li>
+        <li class="navy campaign">
+          <a class="nav-link" href="#">Vacancies </a>
+        </li>
+      </ul>
+      <div class="hamburger">
+        <img src="Img\menu-sharp-custom.svg" alt="Menu">
+      </div>
+    </nav>
+    </header>
+  
+    <section class="main">
+        <!-- Section Headline-->
+        <div class="headline">
+          <div class="wrapper into">
+            <h1>"Welcome to Daa Nkwa!!!"</h1>
+            <h2>You are, we care</h2>
+              <p class="celebration">
+              At Daa Nkwa, our vision is to create an enduring legacy. To achieve this, we are committed to providing our customers with the finest agricultural products and services. Additionally, we strive to offer a serene atmosphere where they can unwind and relax after a long day's work. Our goal is to ensure that every aspect of their experience with us is exceptional, allowing them to thrive in both their professional and personal lives.<strong> Currently we operate mostly in the Ashanti region and ready to move to other parts of the country</strong>
+            </p>
+            <h3>2023.11.99(THU) ~ 27(FRI) 15:00 GMT</h3>
+            <span>Visit us at @ AZ-5, Asankare for our next event session, you reach us via link below for further directions.</span>
+          </div>
+        </div>
+        <section class="main-programs" id="program">
+          <div class="program-title">
+            <h2>Product and Services</h2>
+            <span class="line"></span>
+          </div>
+          <div class="program-cards">
+            <div class="card">
+              <img src="Img/14-141188_farmer-free-transparent-images-farmer-transparent-clipart.png" alt="Lecture">
+              <span>Farms</span>
+              <p class="card-p">We have multi operational areas, you can have a look at where your food comes from</p>
+            </div>
+            <div class="card">
+              <img src="Img/under-construction-vector-png-red-lake-falls-29.jpg" alt="Lecture">
```

Comment:
- Kindly, the product and services section cards have different sizes unlike the Behance Design, so Please update it to have a better UI as the previous reviewer suggested in this [comment](https://github.com/obibaadoma/captone_DaaNkwa/pull/1#discussion_r1246671940) 👍